### ISSUE TITLE
Error when specifying grid of regularization path values in QUICGraphLassoCV

### ISSUE THREAD

**Issue by mnarayan** [skggm/skggm#102]
QUICGraphLassoCV expects self.lams to be a `collections.Sequence`. 
https://github.com/skggm/skggm/blob/develop/inverse_covariance/quic_graph_lasso.py#L575

However specifying the grid of values for path mode is usually a numpy array.  Since the check for a path of values fails, it errors as follows. 

```
/home/user/PYTHON/skggm/inverse_covariance/quic_graph_lasso.py in fit(self, X, y)
    581             lam_0 = 1e-2 * lam_1
    582             path = np.logspace(
--> 583                 np.log10(lam_0), np.log10(lam_1), self.lams
    584             )[::-1]
    585
```
@jasonlaska is there any reason we compare self.lams to collections.Sequence? It seems like path of values is usually an array of floats 
https://github.com/skggm/skggm/blob/develop/inverse_covariance/inverse_covariance.py#L78

**Comment by jasonlaska** [comment]
It's checking whether it's a list, the expectation being that the `lams` passed in would be a list.  I think the right fix would be to invert the condition and instead check for a scalar, which would handle both np.array and a list.

**Comment by mnarayan** [comment]
Yes, I made some changes along these lines. Will push a fix.

**Comment by mnarayan** [comment]
Fixed in PR #101. However, if a list is ever passed subsequent calls to QuicGraphLasso will hit errors. However, there is no reason for someone to pass a list since the interface to QuicGraphLasso asks for an array of floats and similar convention will be expected to follow elsewhere.
